### DISCUSSION TITLE
[designspaceLib] make VariableFontDescriptor for discrete Location?

### DISCUSSION THREAD

**Discussion by LettError** [fonttools/fonttools#2899] [Q&A]
Hypothetical: I have a WIP designspace with some discrete axes, but it does not have any `VariableFontDescriptors` yet. Does designspaceLib have a convenient way of making a placeholder `VariableFontDescriptor` for each discrete location? So that I can (hopefully) pass the designspace to `varLib.build()` to generate all Vfs?

I could use `designspaceLib.split.splitInterpolable` to make separate spaces first, but that seems like a long way round. @belluzj ? thanks!

**Comment by LettError** [comment]
I found `getVariableFonts()` that seems to do that.

**Reply by justvanrossum** [reply]
Cool, I didn't know that either. Very useful!

**Reply by belluzj** [reply]
Yes indeed that's the intent: https://fonttools.readthedocs.io/en/latest/designspaceLib/python.html#fontTools.designspaceLib.DesignSpaceDocument.getVariableFonts
